spec/abstractops.html

```diff
@@ -949,10 +987,10 @@ <h1>ISO 8601 grammar</h1>
           Digits TimeFraction? SecondsDesignator
 
       DurationMinutes :
-          Digits MinutesDesignator DurationSeconds?
+          Digits TimeFraction? MinutesDesignator DurationSeconds?
```

Comment:
I think the grammar should show that the fraction is incompatible with the seconds part:
```
DurationMinutes :
    Digits MinutesDesignator DurationSeconds?
    Digits TimeFraction MinutesDesignator
```

Reply:
What we're doing right now is accept a duration with all the fractions, set the units to zero appropriately and handle them that way. We could change the spec text and instead of throwing at a later time, removing all those checks etc.

Reply:
I am not 100% sure, but I think `PT0.5H30M` is actually illegal according to ISO 8601. In the polyfill it doesn't matter whether we throw because it's not accepted by the regex or we throw right after, but I think the spec grammar should match what's a legal string as much as possible.

Reply:
Correct. Only the smallest unit is permitted to be fractional in ISO 8601.

Reply:
The string still throws, it just throws after the parsing stage (in `DurationHandleFractions`). I thought it was okay as long as we threw on the wrong format.